pkg/machines/libvirt-dbus.js

```diff
@@ -1296,6 +1340,15 @@ export function getLibvirtVersion(connectionName) {
             .then(version => dispatch(updateLibvirtVersion({ libvirtVersion: version[0].v })));
 }
 
+export function getSnapshotsAPIsAvailability(connectionName) {
```

Comment:
Why you want to store that to the store?
It's something that you probably need to check once - and hide the snapshots tab conditionally.
I believe it's much more straightforward to just add this funciton in the component that needs the result of the check.

Reply:
Oh I forgot to make that condition, but snapshots APIs are called apart from inside snapshot tab UI component also in libvirt-dbus.jsx function getVm. So I want to also have a condition there so nonexistent APIs are not called unnecessary.

Reply:
Oh wait, now I see that checking store in getVm in libvirt-dbus.jsx is not really convenient.

Reply:
Okay so I turned vmExpandedContent into stateful component and keep there the state of whetever snapshots are available or not. Hopefully this is effective solution.